File: account-service/internal/models/dto/UserUpdateRequest.go
package dto

// UserUpdateRequest - для обновление сущности пользователя (рассылается остальным сервисам)
type UserUpdateRequest struct {
	Username  string `json:"username"`
	Email     string `json:"email"`
	Password  string `json:"password"`
	EventType string `json:"event_type"` // "CREATE" или "UPDATE"
}

File: notification-service/src/main/java/ru/nand/notificationservice/entity/dto/UserUpdateRequest.java
package ru.nand.notificationservice.entity.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserUpdateRequest {
    private Long id;

    private String username;

    private String email;

    private String password;

    @JsonProperty("event_type")
    private String eventType;
}
